osu.Game/Tests/Visual/OsuGameTestScene.cs

```diff
@@ -106,6 +108,11 @@ protected void PushAndConfirm(Func<Screen> newScreen)
 
         protected void ConfirmAtMainMenu() => AddUntilStep("Wait for main menu", () => Game.ScreenStack.CurrentScreen is MainMenu menu && menu.IsLoaded);
 
+        /// <summary>
+        /// Dismisses any notifications pushed which block from interacting with the game (or block screens from loading, e.g. <see cref="Player"/>).
+        /// </summary>
+        protected void DismissAnyNotifications() => Game.Notifications.State.Value = Visibility.Hidden;
```

Comment:
Aside from being slightly annoyed that borderline unrelated changes are being shipped together with other changes again, I'd like to ask - mostly for clarification purposes - if there was any particular reason to change how this was implemented from the original version?

Reply:
The point of it was dismissing notifications, and I felt off just clicking in game center for such method, so I simplified it instead.

Apologies for the unrelated changes, but wanted to stop from copying that method once again.

Reply:
I think this method is probably more reliable (although it no longer covers the "click to dismiss" flow? not sure).